src/uct/ib/rc/accel/rc_mlx5_struct.h

```diff
@@ -0,0 +1,86 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2016.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCT_RC_MLX5_STRUCT_H
```

Comment:
why need to move the structs from rc_mlx5_common.h to this file?

Reply:
test_obj_size.cc includes DC headers. It can't include entire rc_mlx5_common.h.